rfcs/0005-moxy-carousel.md

```diff
@@ -0,0 +1,195 @@
+- Start Date: 2020-01-21
+- Project: [react-carousel](https://github.com/moxystudio/react-carousel)
+- Reference Issues: [OFICINA-43](https://moxy.atlassian.net/browse/OFICINA-43)
+- Implementation PR: <https://gitlab.com/moxystudio/inoveretail/inovretail-www/merge_requests/102>
+
+# Summary
```

Comment:
Should the `Carousel` component perform lazy load? If we have several carousels on the same page it can get heavy

@acostalima

Reply:
I think it should! @acostalima had already talked to me about this, actually. My bad, I forgot to add this to the discussion. Thanks for pointing this out @ruipneves 👍